packages/dds/tree/README.md

```diff
@@ -1,3 +1,59 @@
 # @fluid-internal/tree
 
 This DDS is not yet ready for public consumption. See (#8273)[https://github.com/microsoft/FluidFramework/issues/8273].
+
+## Motivation
+
+There are a lot of different factors motivating the creation of this Tree DDS.
+A wide variety of possible consumers (across several companies) have overlapping feature requirements
+which seem like they can best be met by collaborating on a single feature rich tree implementation powered by fluid.
+The current feature focus is on:
+
+- Semantics:
+    -   Highly quality semantic merges, including "slice" moves of parts of sequences.
+    -   Transactionality.
+    -   Support schema in a semantically robust way.
+- Scalability:
+    -   Support for partial checkouts: allow efficiently viewing and editing parts of larger datasets without downloading the whole thing.
+    -   Ability to easily (sharing code with client) spin up optional services to improve scalability further (ex: server side summaries, indexing, permissions etc.)
+    -   Efficient data encodings.
+- Expressiveness:
+    -   Efficient support for moves, including moves or large sections of large sequences, and large subtrees.
+    -   Support history operations (ex: undo and redo).
+    -   Flexible schema system that has design patterns for making schema changes over time.
+- Workflows:
+    - Good support for offline.
+    - Optional support for branching and history.
+- Extensibility: future users will have needs beyond what we can afford to support in an initial version and must be practical to accommodate. This includes things like:
+    - New field kinds to allow data-modeling with more specific merge semantics (ex: adding support for special collections like sets, or sorted sequences)
+    - New services (ex: to support permissions, server side indexing etc.)
+
+## Whats missing from existing DDSs?
+
+`directory` and `map` can not provide merge resolution that guarantees well-formedness of trees while supporting the desired editing APIs (like subsequence move),
+and are missing (can can not be practically extended to have) efficient ways to handle large data ort schema.
+
+`sequence` does not capture the hierarchy or schema, and also does not handle partial checkouts.
+Additionally its actual merge resolution leaves some things to be desired in some cases which `tree` aims to improve on.
+
+`expirmental/tree` does not have a built in schema system reducing the data available to make semantically high quality merges.
+It also does merge resolution in a way that does not scale to not having the whole tree in memory due to it being based entirely on node identifiers,
```

Comment:
Don't think this sentence reads very well.

Maybe `It also doesn't do merge resolution in a way that doesn't require having the whole tree in memory due to it being based entirely on node identifiers (including constraints within transactions that can't be verified without reading large parts of the tree).`